src/NUnitFramework/framework/Constraints/PathConstraint.cs

```diff
@@ -117,7 +116,10 @@ protected bool IsSubPath(string path1, string path2)
                 return false;
 
             // path 2 is longer than path 1: see if initial parts match
-            if (!StringUtil.StringsEqual(path1, path2.Substring(0, length1), caseInsensitive))
+            StringComparison comparisonType = caseInsensitive
+                ? StringComparison.CurrentCultureIgnoreCase
+                : StringComparison.CurrentCulture;
```

Comment:
I know this is a literal transpose from the `StringUtil` class, but would wonder if we should use `Ordinal`.
I don't think that file system paths are cultural aware.